### ISSUE TITLE
Issues with Angular 8  while building.

### ISSUE THREAD

**Issue by shan75** [nicky-lenaers/ngx-scroll-to#130]
Hi

I am having issues while compiling to release. It throws up the following error.  In debug mode it seems to work okey

ERROR in ./node_modules/@nicky-lenaers/ngx-scroll-to/fesm2015/nicky-lenaers-ngx-scroll-to.js
Module build failed (from ./node_modules/@angular-devkit/build-optimizer/src/build-optimizer/webpack-loader.js):
TypeError: Cannot read property 'kind' of undefined
    at isAngularDecoratorMetadataExpression (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:230:35)
    at checkNodeForDecorators (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:53:21)
    at visitNodes (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16631:30)
    at Object.forEachChild (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:16859:24) 
    at transformer (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\src\transforms\scrub-file.js:39:16)
    at transformSourceFileOrBundle (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:67384:57)
    at transformation (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:83586:24)      
    at transformRoot (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:83606:82)       
    at Object.map (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:499:29)
    at Object.transformNodes (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:83593:30)
    at emitJsFileOrBundle (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:84089:32)  
    at emitSourceFileOrBundle (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:84043:13)
    at forEachEmittedFile (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:83832:34)  
    at Object.emitFiles (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:84026:9)     
    at emitWorker (D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:89951:33)
    at D:\Development\3DPrinting\Development\3dprintingangular\angularapp\node_modules\@angular-devkit\build-optimizer\node_modules\typescript\lib\typescript.js:89912:66

**Comment by rafa-as** [comment]
@shan75 could you follow the GitHub template to create issues? We need more information to attend your bug.

**Comment by shan75** [comment]
Hello
Thanks for getting back.  I am using Angular 8 so tried to install as per your documentation.  It seems it is down to angular version I got.

Details in this link
https://stackoverflow.com/questions/53827570/typeerror-cannot-read-property-kind-of-undefined-at-getassignmenttargetkind/59949790#59949790

I tried to update my angular version to 'npm i @angular-devkit/build-angular@0.803.7'  after that it works perfectly. This issue is while building for prod. After upgrading my angular version it works Thanks
